logger_test.go

```diff
@@ -0,0 +1,139 @@
+package tracelog
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+)
+
+var TestBasicFormat = "%s [%d] %s: %s"
+var TestBasicFields = []string{"time", "pid", "level", "message"}
+
+func GetFieldValuesForTest(loggerType LoggerType) func() Fields {
+	return func() Fields {
+		now := time.Now().UTC()
+		fields := Fields{
+			"time":  now.Format("2006-01-02 03:04:05 UTC"),
+			"pid":   os.Getpid(),
+			"level": loggerType,
+		}
+
+		return fields
+	}
+}
+
+func TestLogger_PrintWithTextWriter(t *testing.T) {
```

Comment:
Many tests with only little details different, extract the separate function with common code and provide the input / expected output.